niviz_rater/index.py

```diff
@@ -111,31 +113,25 @@ def build_qc_entities(self, image_list):
         return qc_entities
 
 
-def get_qc_bidsfiles(qc_dataset, qc_config, bids_config):
+def build_index(qc_dataset, qc_config, bids_config=None):
     '''
-    Get BIDSFiles associated with qc_dataset
+    Build database
     '''
 
+    if bids_config is None:
+        bids_config = DEFAULT_BIDS
+
     add_config_paths(user=bids_config)
+    bids_configs = bids.config.get_option('config_paths').values()
+
+    config = validate_config(qc_config, bids_configs)
     layout = BIDSLayout(qc_dataset,
                         validate=False,
                         index_metadata=False,
                         config=["user"])
-    bidsfiles = layout.get(extension=qc_config['ImageExtensions'])
-    return bidsfiles
-
 
-def build_index(qc_dataset, qc_config, bids_config=None):
-    '''
-    Build database
-    '''
-
-    if bids_config is None:
-        bids_config = DEFAULT_BIDS
-        print(DEFAULT_BIDS)
+    bidsfiles = layout.get(extension=config['ImageExtensions'])
 
-    config = _validate_config(qc_config)
```

Comment:
It looks like this function can be removed?

Reply:
it's been updated to `validate_config` from the `validation.py` module, it's been removed already :)

Reply:
btw it looks like the git diff mixed up a bunch of changes in a weird way, sorry about that!